jobs/validate/tigera-ee-spec

```diff
@@ -51,7 +75,7 @@ function juju::deploy
 ###############################################################################
 SNAP_VERSION=${1:-1.25/edge}
 SERIES=${2:-focal}
-JUJU_DEPLOY_BUNDLE=kubernetes-tigera-secure-ee
+JUJU_DEPLOY_BUNDLE=charmed-kubernetes
```

Comment:
the bundle `kubernetes-tigera-secure-ee` is not on charmhub -- instead just deploy the above overlay